test/specs/init.js

```diff
@@ -29,6 +29,21 @@ module.exports = function (promise) {
       expect(fsm.current).to.be.equal('green');
     });
 
+    it('should accept an Object for events property', function () {
+      StateMachine.Promise = promise;
+
+      var fsm = StateMachine({
+        events: {
```

Comment:
The events property of the configuration object is array in the other examples. Do you really need `events` as an object?

I have some state machines where this approach will not work as they have multiple events having the same name but starting from different states.